### PR TITLE
Changes the gimlet recipie

### PR DESCRIPTION
**Pull request by Yonnimer**
What it says on the tin. Gimlet currently shares a recipe with gin fizz, causing gin fizz to be impossible to make.

### CODE REVIEW

**Comment by Yonnimer** [comment]
!review

File: html/changelogs/gimlet.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.  
+author: ChangeMe
```
Comment (alsoandanswer) [inline html/changelogs/gimlet.yml:30]: walter

Comment (Yonnimer) [reply html/changelogs/gimlet.yml]: :dab:

File: html/changelogs/gimlet.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - tweak: "Changes the gimlet recipie to 1 gin, 1 limejuice, 1 grenadine."
```
Comment (NonQueueingMatt) [inline html/changelogs/gimlet.yml:41]:
```suggestion
  - tweak: "Changes the gimlet recipe to 1 gin, 1 limejuice, 1 grenadine."
```

**Pull request merged**